src/Core/Grid/Query/OrderQueryBuilder.php

```diff
@@ -122,8 +113,16 @@ public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria)
      */
     public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria)
     {
-        return $this->getBaseQueryBuilder($searchCriteria->getFilters())
-            ->select('COUNT(o.id_order)');
+        $qb = $this->getBaseQueryBuilder($searchCriteria->getFilters());
+        if (isset($searchCriteria->getFilters()['new'])) {
+            $qb = $this->addNewCustomerField($qb->addSelect('o.id_order as o_id_order'));
+            $qb = $this->applyNewCustomerFilter($qb, $searchCriteria->getFilters());
+            $qb->select('count(o_id_order)');
```

Comment:
`$qb->select('count(o_id_order)');` can always be performed after the `if` statement, and the `else` statement can then be removed

Reply:
@jolelievre It's not exactly the same line, one is `o_id_order` the other one is `o.id_order` 😄

Reply:
Ooohhh, my bad! 😅